src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -81,6 +82,31 @@ public Task GetRevisionsFor()
             }
         }
 
+        [RavenAction("/databases/*/revisions/revert", "GET", AuthorizationStatus.ValidUser)]
+        public async Task Revert()
+        {
+            var time = GetDateTimeQueryString("point-in-time");
+            var window = GetTimeSpanQueryString("window", required: false) ?? TimeSpan.FromHours(48);
+
+            var operationId = GetLongQueryString("operationId", required: false) ?? Database.Operations.GetNextOperationId();
+            var token = CreateOperationToken();
+
+            try
+            {
+                await Database.Operations.AddOperation(
```

Comment:
The process may take a *long* time, we want to return the operation id and let the user poll the op state.